setup.py

```diff
@@ -6,7 +6,7 @@
     author="Department for Levelling Up, Housing and Communities",
     description="Provides access to functionality common to creating a data dashboard.",
     name="gov_uk_dashboards",
-    version="4.11.0",
+    version="4.11.1",
```

Comment:
Patch number increased to check tooltip component works on LG dashboard. Package version number to be changed back to 4.11.0 before merging.